lib/core/src/lib/card-view/components/card-view-dateitem/card-view-dateitem.component.stories.ts

```diff
@@ -0,0 +1,128 @@
+/*!
+ * @license
+ * Copyright 2019 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Meta, moduleMetadata, Story } from '@storybook/angular';
+import { CardViewDateItemComponent } from './card-view-dateitem.component';
+import { CoreStoryModule } from './../../../testing/core.story.module';
+import {
+    CardViewDateItemModel,
+    CardViewDatetimeItemModel,
+    CardViewModule
+} from '../../../../..';
+
+export default {
+    component: CardViewDateItemComponent,
+    title: 'Core/Card View/Card View Date Item',
+    decorators: [
+        moduleMetadata({
+            imports: [CoreStoryModule, CardViewModule]
+        })
+    ],
+    argTypes: {
+        editable: {
+            control: 'boolean',
+            description: 'Defines if CardView item is editable',
+            defaultValue: false,
+            table: {
+                type: {summary: 'boolean'},
+                defaultValue: {summary: false}
+            }
+        },
+        displayEmpty: {
+            control: 'boolean',
+            description: 'Defines if it should display CardView item when data is empty',
+            defaultValue: true,
+            table: {
+                type: {summary: 'boolean'},
+                defaultValue: {summary: true}
+            }
+        },
+        displayClearAction: {
+            control: 'boolean',
+            description: 'Defines if it should display clear input action (only with SingleValued components)',
+            defaultValue: true,
+            table: {
+                type: {summary: 'boolean'},
+                defaultValue: {summary: true}
+            }
+        },
+        property: {
+            description: 'Card View Item Model with data',
+            table: {
+                type: { summary: 'CardViewDateItemModel | CardViewDatetimeItemModel' }
+            }
```

Comment:
Inconsistent spacing `type: {summary: 'boolean'}` vs ` type: { summary: 'CardViewDateItemModel | CardViewDatetimeItemModel' }`, but it's a minor problem 😄

Reply:
That's good point! I will lint every file once again to prevent these situations 😅

Reply:
@marcol13 be sure to use `nx run core:lint`. The `nx run lint` only lints demoshell

Reply:
I did it manually - already fixed